packages/components/src/tabs/index.tsx

```diff
@@ -151,13 +151,7 @@ function Tabs( {
 		if ( tabsHavePopulated.current && !! selectedTabId && ! selectedTab ) {
 			setSelectedId( null );
 		}
-	}, [
-		isControlled,
-		selectedId,
```

Comment:
`selectedId` is not referenced inside this hook, and nothing in the history suggests that it was an intentional addition to the dep array. I have to think it was just a mistake.

Reply:
Given that `selectedTab` is explicitly derived from `selectedId`, and `selectedId` isn't used inside the hook, the change works for me.